File: Javascript/8-Kyu/Cat-years-dog-years.js
// Cat years, dog years
var humanYearsCatYearsDogYears = function(humanYears) {

    switch (humanYears) {
        case 1:
            return [humanYears, 15, 15];
        case 2:
            return [humanYears, 24, 24];
        default:
            return [humanYears, (humanYears * 4 + 16), (humanYears * 5 + 14)];
    };

}; 

// Tests
console.log(humanYearsCatYearsDogYears(1));  // --> [1,15,15]
console.log(humanYearsCatYearsDogYears(2));  // --> [2,24,24]
console.log(humanYearsCatYearsDogYears(10));  // --> [10,56,64]

File: Python/8-Kyu/Cat-years-dog-years.py
# Cat years, dog years
def human_years_cat_years_dog_years(human_years):

    years = [human_years, 0, 0]

    if human_years >= 1:
        years = [human_years, 15, 15]
        if human_years >= 2:
            years = [human_years, 24, 24]
            if human_years > 2:
                for x in range(2, human_years):
                    years[1] += 4
                    years[2] += 5

    return years

# Tests
print(human_years_cat_years_dog_years(1))  # --> [1, 15, 15]
print(human_years_cat_years_dog_years(2))  # --> [2, 24, 24]
print(human_years_cat_years_dog_years(10))  # --> [10, 56, 64]
